ui/src/variables/actions/thunks.ts

```diff
@@ -431,6 +435,8 @@ export const selectValue = (variableID: string, selected: string) => async (
   // Validate that we can make this selection
   const vals = normalizeValues(variable)
   if (!vals.includes(selected)) {
+    // TODO: there is an issue that's causing non-state set values to
```

Comment:
This is more of a note-to-self so that we don't lose context on one of the main issues with variable hydration. More specifically, when querying variables from the state that haven't already been set, their values are null. This has an adverse affect on variables who's values are being set in query parameters before they are actually loaded by the app